### DISCUSSION TITLE
Fast sign in by a link with temporary token

### DISCUSSION THREAD

**Discussion by rtmbox** [nextauthjs/next-auth#3968] [Help]
I want to find a way to create a get link for a quick login.
For example domain.com/auth/{token} where {token} is some known token associated with the user.
How can I create an authorization session by processing this route?

**Comment by rtmbox** [comment]
How can I make server-side authorization in my backend?

**Reply by 0ubbe** [reply]
Could you explain a bit more your use case? I don't understand the auth flow you want to achieve. You can also point to an article explaining it if that's easier 👍🏽

Have you had a look at the [Email Provider](https://next-auth.js.org/providers/email)? It seems similar to what you want to achieve. Once set up, the user will receive a **magic link** on their email to sign in, similar to apps like [Notion](notion.so/) do.
